### PR TITLE
A tutorial using simulated Gaussian data

### LINKED ISSUE #131: Add support for Gaussian data modelling in Priors shiny app
The current version of our Priors Shiny app only supports Poisson data modelling. To enhance its versatility and utility, we need to extend its capabilities to accommodate Gaussian data modelling as well. 

Xueqing

### PR DESCRIPTION
**Pull request by XueqingYin**
* **Summary of changes** (Bug fix, feature, docs update, ...)

This is to create a tutorial that covers simulating Gaussian data, fitting the BHM and exploring the key functionality in fdmr.

* **Please check if the PR fulfills these requirements**

- [ ] Closes #131 #112 
- [ ] Tests added and passed
- [ ] All code checks passing - `styler` run over code
- [x] Vignettes updated/added
- [ ] Added an entry in the latest `CHANGELOG.md` file if fixing a bug or adding a new feature
- [ ] Any new libraries added to `DESCRIPTION`

### CODE REVIEW

File: vignettes/Simulation_GaussianDat.Rmd [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                  LAT = rep(locs[,2],each=n.time))
+
+```
+
+A histogram is created to visualize the distribution of the simulated Gaussian data.
+
+```{r hist,error=TRUE,fig.cap="A histogram of the simulated Gaussian data.", fig.width=8, fig.height=4, fig.align = "center"}
+hist(simdf$datn,main='', xlab='Data')
+```
+
```
Comment (mnky9800n) [inline vignettes/Simulation_GaussianDat.Rmd:125]: i think here you should put a map of the data too

**Review by mnky9800n — CHANGES_REQUESTED** [review]
i think you should have a map of the raw data before you start modeling the data.

**Review by mnky9800n — APPROVED** [review]
love the map

**Comment by mnky9800n** [comment]
```
Caused by error:
! Invalid dataset, the folder /home/runner/fdmr/tutorial_data/priors does not exist.
Backtrace:
    ▆
 1. └─fdmr::load_tutorial_data(dataset = "priors", filename = "spatial_dataBris.rds")
 2.   └─fdmr::get_tutorial_datapath(dataset = dataset, filename = filename)
Execution halted
Error: Process completed with exit code 1.
```

**Comment by XueqingYin** [comment]
@mnky9800n That's strange. I have no issues running ```sp_data <- fdmr::load_tutorial_data(dataset = "priors", filename = "spatial_dataBris.rds")```.  "spatial_dataBris.rds" is saved in the fdmr_data repo in "priortut_data.tar.bz2 (https://github.com/4DModeller/fdmr_data/blob/main/priortut_data.tar.bz2). 

Xueqing

**Comment by XueqingYin** [comment]
@mnky9800n  I know why it happens. I should have retrieved the priors data first before loading in the data. It's now fixed. I added a line of code that uses the ```fdmr::retrieve_tutorial_data``` function to retrieve the data first from fdmr data store before loading. Could you please try again and let me know if it works?

Xueqing

**Pull request merged**